intel/AiCSD
# fix: updated testcontainers and docker in go.mod, resolves dependabot vulnerabilities

## PR Checklist

## Author Mandatory (to be filled by PR Author/Submitter)

### CODE MAINTAINABILITY

    - <https://github.com/intel/AiCSD/blob/main/.github/CONTRIBUTING.md>

### What are the steps to test this PR contribution?

How does a reviewer know it works?
 1. Check that only docker compose v2 is installed with `docker compose version` 
 2. Run `make integration-test` ... observe no errors."

Are there any special instructions beyond standard workflow?



## Maintainer Mandatory (to be filled by PR Reviewer/Approving Maintainer)

### QUALITY CHECKS


## Other information
